api/services/GithubBuildStatusReporter.js

```diff
@@ -77,10 +77,10 @@ const reportBuildStatus = (build) => {
   let options = {};
 
   return new Promise((resolve, reject) => {
-    if (!build || !build.commitSha) {
-      reject(new Error('Build or commit sha undefined. Unable to report build status'));
-    } else {
+    if (build && (build.webhookCommitSha || build.cloneCommitSha)) {
```

Comment:
Is there a scenario when we would have `cloneCommitSha` without `webhookCommitSha` ?

Reply:
point taken